src/Cli.Tests/ConfigureOptionsTests.cs

```diff
@@ -246,6 +246,165 @@ public void TestUpdateMultipleParametersForGraphQLSettings()
+            // Arrange -> all the setup which includes creating options.
+            SetupFileSystemWithInitialConfig(INITIAL_CONFIG);
+
+            // Act: Attempts to update enabled flag
+            ConfigureOptions options = new(
+                runtimeRestEnabled: updatedEnabledValue,
+                config: TEST_RUNTIME_CONFIG_FILE
+            );
+            Assert.IsTrue(TryConfigureSettings(options, _runtimeConfigLoader!, _fileSystem!));
+
+            // Assert: Validate the Enabled Flag is updated
+            string updatedConfig = _fileSystem!.File.ReadAllText(TEST_RUNTIME_CONFIG_FILE);
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(updatedConfig, out RuntimeConfig? runtimeConfig));
+            Assert.IsNotNull(runtimeConfig.Runtime?.Rest?.Enabled);
+            Assert.AreEqual(updatedEnabledValue, runtimeConfig.Runtime.Rest.Enabled);
+        }
+
+        /// <summary>
+        /// Tests that running "dab configure --runtime.rest.path" on a config with various values results
+        /// in runtime config update. Takes in updated value for rest.path and 
+        /// validates whether the runtime config reflects those updated values
+        [DataTestMethod]
+        [DataRow("/updatedPath", DisplayName = "Update path to /updatedPath for Rest.")]
+        [DataRow("/updated_Path", DisplayName = "Ensure underscore is allowed in Rest path name.")]
+        [DataRow("/updated-Path", DisplayName = "Ensure hyphen is allowed in Rest path name.")]
+        public void TestUpdatePathForRestSettings(string updatedPathValue)
+        {
+            // Arrange -> all the setup which includes creating options.
+            SetupFileSystemWithInitialConfig(INITIAL_CONFIG);
+
+            // Act: Attempts to update path value
+            ConfigureOptions options = new(
+                runtimeRestPath: updatedPathValue,
+                config: TEST_RUNTIME_CONFIG_FILE
+            );
+            Assert.IsTrue(TryConfigureSettings(options, _runtimeConfigLoader!, _fileSystem!));
+
+            // Assert: Validate the Path update is updated
+            string updatedConfig = _fileSystem!.File.ReadAllText(TEST_RUNTIME_CONFIG_FILE);
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(updatedConfig, out RuntimeConfig? runtimeConfig));
+            Assert.IsNotNull(runtimeConfig.Runtime?.Rest?.Path);
+            Assert.AreEqual(updatedPathValue, runtimeConfig.Runtime.Rest.Path);
+        }
+
+        /// <summary>
+        /// Tests that running "dab configure --runtime.rest.request-body-strict" on a config with various values results
+        /// in runtime config update. Takes in updated value for rest.request-body-strict and 
+        /// validates whether the runtime config reflects those updated values
+        [DataTestMethod]
+        [DataRow(false, DisplayName = "Update request-body-strict to be false for Rest.")]
+        [DataRow(true, DisplayName = "Update request-body-strict to be true for Rest.")]
```

Comment:
```suggestion
        [DataRow(false, DisplayName = "Update request-body-strict from true to false.")]
        [DataRow(true, DisplayName = "Update request-body-strict from false to true.")]
```

-> Update request-body-strict from false to true. may need more changes to the test.